components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -6913,13 +6814,6 @@ public void deletePolicyGroup(String applicationId, String policyGroupId) throws
 				log.debug("Policy Group deleted successfully. " + strDataContext);
 			}
 		} catch (SQLException e) {
-			if (conn != null) {
-				try {
-					conn.rollback();
-				} catch (SQLException e1) {
-					log.error("Failed to rollback while deleting the policy group" , e);
-				}
-			}
 			String strDataContext =
```

Comment:
This line break automatically get applied when we format the code with WSO2 code formatter.